code/modules/mob/living/simple_mob/subtypes/animal/hyena/hyena.dm

```diff
@@ -0,0 +1,158 @@
+/*
+
+Hyenas are any feliform carnivoran mammals of the family Hyaenidae. With only four extant species (in three genera), it is the fifth-smallest biological family in the Carnivora, and one of the smallest in the class Mammalia.
+They're also cool, and Rykka/Nyria wrote this uwu
+
+*/
+
+// Cataloguer data here
+/datum/category_item/catalogue/fauna/hyena
+	name = "Desert Fauna - Hyena"
+	desc = "Classification: Crocuta crocuta \
+	<br><br>\
+	It is primarily a hunter but may also scavenge, with the capacity to eat and digest skin, bone and other animal waste. \
+	The spotted hyena displays greater plasticity in its hunting and foraging behaviour than other similar carnivores; \
+	it hunts alone, in small parties of 2–5 individuals or in large groups. \
+	During a hunt, spotted hyenas often run through ungulate herds in order to select an individual to attack. \
+	Once selected, their prey is chased over a long distance, often several kilometres, at speeds of up to 60 km/h."
+	value = CATALOGUER_REWARD_MEDIUM
+
+// Start Defining the mob here
+/mob/living/simple_mob/animal/hyena 
+	name = "Hyena"
+	desc = "Yeen! Its fur is a dusty yellow-brown color with black spots, and it has rounded ears... and SHARP TEETH! You inexplicably want to pet it, though."
+	tt_desc = "Crocuta crocuta"
+	catalogue_data = list(/datum/category_item/catalogue/fauna/hyena)
+
+	icon_state = "hyena"
+	icon_living = "hyena"
+	icon_dead = "hyena_dead"
+	has_eye_glow = FALSE // Change this to true for fun spooky eye glow in darkness. <3
+	minbodytemp = 175 // Make hyenas able to survive freezing cold. Someone criticize me later uwu
+	faction = "yeen" // gon fight any other mobs. grr
+	maxHealth = 125 // not as tanky as a spider for obvious reasons, but not a pushover.
+	health = 125
+	pass_flags = PASSTABLE
+	movement_cooldown = 0.25 // Yeen go fast
+	// movement_sound = null - TODO: find good animal pawb sounds
+	poison_resist = 0.1
+
+	see_in_dark = 25 // Hyenas are nocturnal, they should be able to see anything in the dark. <3
+
+	response_help = "rubrubs"
+	response_disarm = "boops aside"
+	response_harm = "smacks"
+
+	melee_damage_lower = 12 // hyenas go for the legs/lower bits
+	melee_damage_upper = 8 // not as high damage, but faster bites. nomnomnom <3
+	attack_sharp = 1
+	attack_edge = 1
+	base_attack_cooldown = 0.5
+	attacktext = list("bit", "nipped", "chomped", "clawed", "scratched", "lewded")
+	attack_sound = 'sound/weapons/bite.ogg' // placeholder till I find a better bite
+
+	ai_holder_type = /datum/ai_holder/simple_mob/melee/pack_mob // Define here what type of enemy hyena is.
+
+	heat_damage_per_tick = 20
+	cold_damage_per_tick = 20
+
+	speak_emote = list("yaps")
+
+	say_list_type = datum/say_list/hyena

```

Comment:
Hmm.... This is still throwing an undefined var issue....

Reply:
Oh, add a front slash.

Reply:
Literally found it just as you did it <3